examples/src/main/java/com/mapbox/navigation/examples/core/FeedbackButtonActivity.kt

```diff
@@ -216,6 +238,10 @@ class FeedbackButtonActivity : AppCompatActivity(), OnMapReadyCallback,
         }
 
         feedbackButton.addOnClickListener {
+            mapboxMap?.locationComponent?.lastKnownLocation?.let {
+                navigationMapboxMap?.addCustomMarker(SymbolOptions().withGeometry(it.toPoint()).withIconImage(iconImageFeedback))
```

Comment:
Adds the custom marker when click `FeedbackButton` and clear all the markers when `onFinalDestinationArrival` gets called.

Reply:
👍